src/views/Game/AIReview.tsx

```diff
@@ -952,13 +966,35 @@ export class AIReview extends React.Component<AIReviewProperties, AIReviewState>
             </div>
         );
     }
-    public renderWorstMoveList(): JSX.Element {
-        if (!this.props.game.goban?.engine?.move_tree || !this.ai_review) {
+
+    public getWorstMoveList(): AIReviewWorstMoveEntry[] {
+        const move_tree = this.props.game.goban.engine.move_tree;
+        let worst_moves: AIReviewWorstMoveEntry[];
+        let threshhold: number;
+
+        if (this.ai_review?.scores) {
```

Comment:
Perhaps more of this logic should live in `Goban::AIReview::computeWorstMoves` (defaulting to score, falling back on winrate, filtering on threshold). @anoek LMK what you think, given that you'd need to make the corresponding backend changes.

Reply:
Yeah that would be great. The backend uses that same code, so if you update it there it'll sync up nicely.

Reply:
Sounds good, I'll make that update later today